### PR TITLE
Limit #features h1.druk font-size

### PR DESCRIPTION
**Pull request by NunoAlexandre**
It becomes too big on ultra-wide screens.

Currently: 
Screenshot from 2020-11-30 11-25-14

With fix:

fix

### CODE REVIEW

**Comment by NunoAlexandre** [comment]
Also fixed the short line-height for this element on bigger screens :point_up:

**Comment by juliendonck** [comment]
Screen Shot 2020-11-30 at 12 25 55
still cut off

**Comment by NunoAlexandre** [comment]
@juliendonck can you have another look?

**Review by brandonhaslegs — DISMISSED** [review]

**Review by juliendonck — CHANGES_REQUESTED** [review]
Screen Shot 2020-12-03 at 12 34 57
Still cut for me ?

**Comment by juliendonck** [comment]
> Screen Shot 2020-12-03 at 12 34 57
> 
> Still cut for me ?

On my 27 inch screen

**Comment by NunoAlexandre** [comment]
@juliendonck you are right. Apologies for the noise and time taken. Pushed a fix, checked it for several screen sizes.

@brandonhaslegs was there a design intent with setting the line-height to < 100% or was it just leftovers?

**Comment by brandonhaslegs** [comment]
> @brandonhaslegs was there a design intent with setting the line-height to < 100% or was it just leftovers?

Can you comment this on the style you're referring to? Don't know which one you mean.

**Comment by NunoAlexandre** [comment]
> > @brandonhaslegs was there a design intent with setting the line-height to < 100% or was it just leftovers?
> 
> @NunoAlexandre Can you comment this on the style you're referring to? Don't know which one you mean.

On the one affected here, the `.druk` class.

**Comment by brandonhaslegs** [comment]
You could remove it but then you need to add a line-height style to the footer `h1`. All other druk headlines already have `line-height` applied separately.

File: docs/styles/style.css [hunk 1]
```diff
@@ -453,7 +453,6 @@ nav a:hover span {
 @media only screen and (max-width: 30rem) {
   #features h1.druk {
     font-size: 28vw;
-    line-height: 80%;
```
Comment (brandonhaslegs) [inline docs/styles/style.css:456]: This changes the line height on mobile, which seems unrelated to the issue.

**Review by brandonhaslegs — APPROVED** [review]

**Pull request merged**